version/1.0.0/README.md

```diff
@@ -23,64 +20,76 @@
 <h1 align="center">v1.0.0</h1>
 
 ## Documentation
+
 You can view the Documentation for Markdown Tag [here](https://marketingpipeline.github.io/Markdown-Tag/1.0.0/docs/landing) or by viewing the html file at `docs/landing`.
-## Example and usage
+
+## Example and Usage
+
 You can view a demo of Markdown Tag in use [here](https://marketingpipeline.github.io/Markdown-Tag).
 
+### Installation
```

Comment:
A quick search shows that "instal" is used in the UK, so I'm not sure if this needs to be changed.